github/client.go

```diff
@@ -420,50 +420,79 @@ func (client *Client) ForkRepository(project *Project) (repo *octokit.Repository
 	return
 }
 
-func (client *Client) Issues(project *Project) (issues []octokit.Issue, err error) {
-	url, err := octokit.RepoIssuesURL.Expand(octokit.M{"owner": project.Owner, "repo": project.Name})
+type IssueParams struct {
```

Comment:
Can we document those types? for instance if you have a reference to the structure that it's reading, that would be awesome: it would auto-document each field, and add a big hint to anybody wanting to extend them.